harvester_e2e_tests/conftest.py

```diff
@@ -421,3 +449,18 @@ def pytest_html_results_table_header(cells):
 
 def pytest_html_results_table_row(report, cells):
     cells.insert(1, f'<td class="col-time">{datetime.utcnow()}</td>')
+
+
+def pytest_report_header(config):
```

Comment:
I don't think it is a good idea to expose sensitive data, it is not necessary for the report.